canonical/ubuntu.com
# Add fieldset sections to fix missing payload on dynamic forms

**Note: all affected forms are using the same design as the dynamic form in [pricing-infra.html](https://github.com/canonical/ubuntu.com/blob/f35ba7aa510f5b0bee8ebfa7493f98394656936b/templates/shared/forms/interactive/pricing-infra.html). 
Only difference is the required fields in the contact section thus only QA review is needed** 

## Done
- Apply redesign on dynamic forms
- Add fieldset sections that adhere with UX and fix missing payload 
- Make sure that the required fields in the forms are similar to Marketo


## QA
- Go to each of the affected forms below
- Fill up the form, submit and check that the payload consolidates form inputs correctly
- Check that the form is submitted to Marketo correctly

Affected forms:
- [x] [/aws#get-in-touch](https://ubuntu-com-14133.demos.haus/aws#get-in-touch)
- [x] [/ceph/consulting#get-in-touch](https://ubuntu-com-14133.demos.haus/ceph/consulting#get-in-touch)
- [x] [/landscape/pricing#get-in-touch](https://ubuntu-com-14133.demos.haus/landscape/pricing#get-in-touch)
- [x] [/landscape/features#get-in-touch](https://ubuntu-com-14133.demos.haus/landscape/features#get-in-touch)
- [x] [/desktop/organisations#get-in-touch](https://ubuntu-com-14133.demos.haus/desktop/organisations#get-in-touch)
- [x] [/security/esm#get-in-touch](https://ubuntu-com-14133.demos.haus/security/esm#get-in-touch)

## Issue / Card

Fixes [WD-13791](https://warthogs.atlassian.net/browse/WD-13791)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-13791]: https://warthogs.atlassian.net/browse/WD-13791?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
